connectors/src/connectors/slack/bot.ts

```diff
@@ -251,15 +253,16 @@ async function botAnswerMessage(
         break;
       }
       case "agent_generation_success": {
-        const finalAnswer = `${_removeCiteMention(
-          event.text
-        )}\n\n <${DUST_API}/w/${connector.workspaceId}/assistant/${
-          conv.conversation.sId
-        }|Continue this conversation on Dust>`;
+        const agentMessage = convRes.value.agentMessage;
```

Comment:
I'm not sure I understand this line? Where is this convRes coming from and how can it hold an up to date agentMessage?

Reply:
Oh boy I see it now.

Let's remove the 10x getConvefrsation :scream:

Reply:
Probably another PR :/

Reply:
Crated https://github.com/orgs/dust-tt/projects/3/views/1?pane=issue&itemId=40134811